### PR TITLE
Update UI/event detail to development branch

### PR DESCRIPTION
**Pull request by JackyDo2005**
Update UI/event detail to development branch

### CODE REVIEW

File: app/events/(components)/card.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+const EventCard: React.FC<CardProps> = ({
+	title,
+	description,
+	date,
+	imageUrl,
+}) => {
+	return (
+		<div className="bg-ft-background-popup bg-opacity-20 transition-shadow duration-300 rounded-2xl w-11/12 h-[500px] relative flex g-5 hover:shadow mb-5 border-gray-300 border-2">
+			<Image
```
Comment (Hankaji) [inline app/events/(components)/card.tsx:26]: Add lazy loading option

File: app/events/(components)/style.css
Comment (Hankaji) [inline app/events/(components)/style.css:1]: Please put your custom style.css inside the folder "styles" located at root folder. Of course inside their specific folder name or just rename this to this specific target (like events-detail.css for example)

File: app/page.tsx
Comment (Hankaji) [inline app/page.tsx:1]: Why put the <Event /> here? this page.tsx is for home page route. Consider putting it in app/events/page.tsx (for events route)

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "bunTest": "bun test --coverage --bail"
   },
   "dependencies": {
+    "@nextui-org/card": "^2.0.28",
```
Comment (Hankaji) [inline package.json:16]: No need to install @nextui-org/card. The @nextui-org/react already contain the card elements. That will cause duplication in dependencies.

**Review by Hankaji — CHANGES_REQUESTED** [review]
Please address the feedback and make proper changes then commit or create another PR for me.

File: app/events/(components)/card.tsx (hunk 1 above)
Comment (JackyDo2005) [reply app/events/(components)/card.tsx:26]: suree, gonna implement it a ho

**Pull request closed without merging**